### DISCUSSION TITLE
Collections of similar Blocks

### DISCUSSION THREAD

**Discussion by petomka** [Minestom/Minestom#1063] [Help] [answered]
Does Minestom feature collections of similar blocks, e.g. for all banners, wall banners, planks, etc.?
This would make developing easier when trying to modify groups of these blocks that share common properties without having to rely on the namespaced key and iterate through every available block.
If there are better alternatives to what I just described, I'd be happy to hear them.

**Comment by MrlnHi** [comment, marked as answer]
block tags

**Reply by petomka** [reply]
> block tags

Please elaborate?

**Reply by MrlnHi** [reply]
minecraft has block tags. so does minestom. you can check if a block is included in a block tag using 
```java
private static boolean isReplaceable(final Block block) {
    var tag = MinecraftServer.getTagManager().getTag(BasicType.BLOCKS, "minecraft:banners");
    if (tag == null) return false;
    return tag.getValues().contains(block.namespace());
}
```

**Reply by MrlnHi** [reply]
minecraft:banners is a block tag that includes all banners and wall banners
